app/posts/register-trainee-teachers/2023-01-31-asking-providers-to-sign-off-trainee-data-for-the-previous-academic-year.md

```diff
@@ -0,0 +1,136 @@
+
+- were in their final year and gained qualified teacher status (QTS) or early years teacher status (EYTS)
+- got a job in a state school within 16 months of gaining QTS
+
+We decided to ask providers to check data for all their trainees for the 2021 to 2022 academic year, not just those who finished their courses in that year. This is not necessary for the performance profiles but will help make sure that trainee data is accurate.
+
+## What we changed
+
+We added:
+
+- guidance for users
+- a report containing the data which users need to check
+
+We decided to split the guidance and report rather than having both on a single page because:
+
+- it allowed us to keep each page short and focused
+- the people who download the report may not be the same ones who sign off the data
+- the service already has separate sections for guidance and reports
+
+The guidance and the report link to each other. When we sent emails to providers about signing off data, we included links to both pages.
+
+We also improved the pages which list guidance pages and reports.
+
+### Guidance
+
+The new guidance page tells providers how to:
+
+- find the data they need
+- check the data and fix any mistakes
+- sign off their data
+
+To find the data, providers can use either:
+
+- the new report we created
+- their list of registered trainees, which can be [filtered to show the previous academic year](/register-trainee-teachers/identifying-trainees-from-a-single-year/)
+
+For each trainee, the provider needs to check:
+
+- course details
+- status
+- the date they were awarded QTS or EYTS, or deferred or withdrew from the course - unless they’re still studying
+
+The guidance gives instructions for fixing mistakes. How users do this depends on whether:
+
+- they’re a provider of school centred initial teacher training (SCITT) or a higher education institution (HEI)
+- the trainee is shown in the service as awarded or withdrawn, or actively training or deferred
+
+Providers sign off the trainee data using a Google form. This should be done by a senior person and not by the same person who entered the data.
+
+We decided that it was not necessary to build a journey within Register for signing off data. People who sign off data may not be users of the service, so we cannot require them to sign in to it.
+
+### Report
+
+We created a new report which includes all trainees who:
+
+- have been registered with the Department for Education
+- studied in the 2021 to 2022 academic year
+
+The report can be downloaded as a CSV file. It contains the same information about each trainee as the normal data export.
+
+The format of the file name is:
+
+YYYY-MM-DD_HH-MM-SS_AAAA-BBBB-trainees_performance-profiles-sign-off_register-trainee-teachers
```

Comment:
Should this have a file extension?
```suggestion
`YYYY-MM-DD_HH-MM-SS_AAAA-BBBB-trainees_performance-profiles-sign-off_register-trainee-teachers`
```

Reply:
I'll add a file extension as it makes it clearer.